runtime/integration-tests/src/generic/mod.rs

```diff
@@ -60,6 +61,8 @@ macro_rules! test_for_runtimes {
             $(
                 #[tokio::test]
                 async fn $runtime_name() {
+                    crate::utils::logs::init_logs();
```

Comment:
Do we really want this for all tests by default? If it's for debugging purposes, why not add it during the debugging phase for the test that is currently failing?

Reply:
I think having it here makes sense and one can tweak the log level when debugging tests individually, if needed.

Reply:
Ok, my comment comes mainly from [this issue](https://github.com/centrifuge/centrifuge-chain/actions/runs/6864028905/job/18664932956?pr=1606), where the amount of stdout increases the CI time until the timeout.

I think successful tests should not give any output (nobody looks logs in successful tests), only the ones that fail. Not sure how to configure the logger to act in that way and also reduce the amount of stdout in CI.

Reply:
Oh, yea, that makes sense. Well, we can set the default log level to error and then the logs would be minimal, I still think it's good to have them enabled by default so that it makes or lives easier when debugging and not having to look for the init func and add it in there manually, but then again, no strong feelings on this one.